docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/pagination/pagination.scss

```diff
@@ -50,7 +50,7 @@
   }
 
   #{$root}__link {
-    width: civictheme-space(3);
+    min-width: civictheme-space(3);
```

Comment:
We use this for the first, next, previous, last links which need to be variable width.